Pipelines/Templates/deploy-Solution.yml

```diff
@@ -134,18 +134,18 @@ steps:
   displayName: 'Get managed solution zip path'
   condition: and(succeeded(), ne('${{parameters.importUnmanaged}}', 'true'))
 
-- pwsh: |    
-    # If Solution Upgrade was not set at Canvas App Deployment Settings screen, check whether PR has Tag.
-    if("$(TriggerSolutionUpgrade)".Contains("TriggerSolutionUpgrade"))
-    {
-        . "$env:POWERSHELLPATH/set-trigger-solution-upgrade.ps1"
-        Set-TriggerSolutionUpgrade '$(TriggerSolutionUpgrade)' '$(System.TeamFoundationCollectionUri)' '$(System.TeamProjectId)' '$(Build.Repository.Name)' '$(Build.Repository.Provider)' '$(Build.SourceVersion)' '$(Build.Reason)' '$(GitHubRepo)' '$(GitHubPAT)'
-    }
-  env:
-    SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-  name: setTriggerSolutionUpgrade
-  condition: and(succeeded(), ne('${{parameters.importUnmanaged}}', 'true'))
-  displayName: 'Set TriggerSolutionUpgrade Variable'
+#- pwsh: |    
+#    # If Solution Upgrade was not set at Canvas App Deployment Settings screen, check whether PR has Tag.
```

Comment:
Need to uncomment this?

Reply:
@mikefactorial This PR is work in progress. My bad I requested review. Yes, this needs to be uncommented.